FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -22,6 +22,10 @@ import Foundation
 @objc(FIRPhoneAuthProvider) open class PhoneAuthProvider: NSObject {
   /// A string constant identifying the phone identity provider.
   @objc public static let id = "phone"
+  @objc private static let kRecaptchaVersion = "RECAPTCHA_ENTERPRISE"
```

Comment:
`private` should not need to be `@objc`

Reply:
Addressed in https://github.com/firebase/firebase-ios-sdk/pull/14170